src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/LanguageServer/RazorDiagnosticsBenchmark.cs

```diff
@@ -48,7 +48,7 @@ public async Task SetupAsync()
         DocumentPullDiagnosticsEndpoint = new DocumentPullDiagnosticsEndpoint(
             languageServerFeatureOptions: languageServer.GetRequiredService<LanguageServerFeatureOptions>(),
             translateDiagnosticsService: languageServer.GetRequiredService<RazorTranslateDiagnosticsService>(),
-            languageServer: new ClientNotifierService(BuildDiagnostics(N)));
+            languageServer: new ClientNotifierService(BuildDiagnostics(N)), null!);
```

Comment:
This will throw since you added code to throw an exception in `DocumentPullDiagnosticsEndpoint` if this value is null.